source/MaterialXContrib/Handlers/SampleObjLoader.cpp

```diff
@@ -10,11 +10,11 @@
 #include <sstream>
 #include <fstream>
 #include <numeric>
-#include <iomanip>  
```

Comment:
Lots of clang-format noise. The MaterialXCore files are less noisy and will show only 2 diff per file.